background.ts

```diff
@@ -99,6 +102,12 @@ mainEvents.on('settings-update', async(newSettings) => {
     pathManager = getPathManagerFor(newSettings.pathManagementStrategy);
     await pathManager.enforce();
   }
+
+  if (newSettings.kubernetes.containerEngine === settings.ContainerEngine.MOBY) {
+    await dockerDirManager.ensureDockerConfig(!newSettings.kubernetes.suppressSudo);
```

Comment:
What does `suppressSudo` mean on Windows?

Reply:
It's set to `false` by default. Though it shouldn't matter any more once I put this call in `WSLBackend`.